fastq_filtrator.py

```diff
@@ -0,0 +1,98 @@
+
+# ### Функция для прочтения файла по 4 строки:
+
+def read_4lines(file):
+    lines = []
+    for i in range(4):
+        line = file.readline()
+        if line == "":
+            return []
+        lines.append(line.replace("\n", ""))
```

Comment:
```suggestion
        lines.append(line.strip())
```
У нас для этого есть специальная функция, но по факту разницы особо нет в данном случае